symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/parsing/Cashtag.java

```diff
@@ -0,0 +1,21 @@
+package com.symphony.bdk.core.activity.parsing;
+
+import lombok.Value;
+import org.apiguardian.api.API;
+
+/**
+ * Class representing a cashtag in a V4Message
+ */
+@API(status = API.Status.INTERNAL)
```

Comment:
Are they really `INTERNAL`? I think we expose them to the developers, let's make them `STABLE` instead